engines/basic_resources/app/controllers/basic_resources/api/v1/memberships_controller.rb

```diff
@@ -22,6 +23,12 @@ def show
         # POST /api/v1/memberships
         #
         def create
+          group = Group.find_by_id(membership_params[:basic_resource_group_id])
```

Comment:
Yep, I believe the inconsistency would be even more annoying and it would lead to bugs